Source/WebCore/loader/cache/CachedFontLoadRequest.h

```diff
@@ -107,7 +107,7 @@ class CachedFontLoadRequest final : public FontLoadRequest, public CachedFontCli
 
         m_fontLoadedProcessed = true;
         ASSERT_UNUSED(font, &font == m_font.get());
-        if (CheckedPtr client = m_fontLoadRequestClient.get())
+        if (WeakPtr client = m_fontLoadRequestClient)
             client->fontLoaded(*this); // fontLoaded() might destroy this object. Don't deref its members after it.
```

Comment:
It seems like we would want to ensure that the `m_fontLoadRequestClient` is alive as long as we are in the `fontLoaded` call. Since the comment says that `fontLoaded` might destroy this object, we might need to be holding the current `this` object until the call returns:

```
if (CheckedPtr client = m_fontLoadRequestClient.get()) {
    RetainPtr protectedThis { this }
    client->fontLoaded(*this); // fontLoaded() might destroy this object. Don't deref its members after it.
}
```

Reply:
Oh! Seems like we can close this patch, then, @cdumez!